.github/workflows/pr-test.yml

```diff
@@ -12,10 +12,18 @@ on:
 jobs:
   build:
     runs-on: macos-latest
+    concurrency:
+      group: build-${{ github.head_ref }}
+      cancel-in-progress: true
     steps:
     - name: Display runner context
       run: hostname -s
-    - uses: actions/checkout@v2
+    - name: Display environment
+      run: echo $GITHUB_BASE_REF $GITHUB_HEAD_REF $GITHUB_HEAD_REF ${GITHUB_BASE_REF##*/} ${GITHUB_HEAD_REF##*/}
```

Comment:
GITHUB_HEAD_REF is twice

Reply:
ahah will have to remove it anyways thanks